Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -350,6 +350,19 @@ public static MixedRealityToolkitModuleType GetModuleFromPackageFolder(string pa
             return moduleNameMap.TryGetValue(packageFolder, out moduleType) ? moduleType : MixedRealityToolkitModuleType.None;
         }
 
+        /// <summary>
+        /// Creates the MixedRealityToolkit.Generated folder if it does not exist and returns the 
+        /// path to the generated folder.
+        /// </summary>
+        public static string GetGeneratedFolder
+        {
+            get
+            {
+                TryToCreateGeneratedFolder();
+                return MapModulePath(MixedRealityToolkitModuleType.Generated);
+            }
+        }
+
```

Comment:
The fix is confirmed, but this property may not be necessary but it could be useful.